### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by Roms1383**
While doing the other PR yesterday I noticed the project wouldn't compile properly probably due to latest changes in RustC, this PR fixes it.

### CODE REVIEW

**Comment by Roms1383** [comment]
As a side note, it seems we fall behind RED4ext.SDK lately, and not sure how to address it. Could you provide some feedback / guidance @jac3km4 ?

File: red4ext-sys/src/lib.rs [hunk 1]
```diff
@@ -31,6 +31,7 @@ pub mod ffi {
         type CStackFrame;
         type CProperty;
         type PluginInfo;
+        #[allow(dead_code)]
```
Comment (jac3km4) [inline red4ext-sys/src/lib.rs:34]:
```suggestion
        #[allow(unused)]
```

File: red4ext/src/conv.rs [hunk 2]
```diff
@@ -219,7 +219,15 @@ where
     A::Repr: Default,
 {
     let mut init = A::Repr::default();
-    unsafe { ffi::get_parameter(frame, std::mem::transmute(&mut init)) };
+    unsafe {
+        ffi::get_parameter(
+            frame,
+            std::mem::transmute::<
+                &mut <A as crate::conv::FromRepr>::Repr,
```
Comment (jac3km4) [inline red4ext/src/conv.rs:226]:
```suggestion
                &mut A::Repr,
```

File: red4ext/src/conv.rs [hunk 3]
```diff
@@ -219,7 +219,15 @@ where
...
+                red4ext_sys::interop::VoidPtr,
```
Comment (jac3km4) [inline red4ext/src/conv.rs:227]: add an import if it's missing
```suggestion
                VoidPtr,
```

File: red4ext/src/invocable.rs [hunk 4]
```diff
@@ -152,7 +152,11 @@ where
     }
 
     let mut ret = R::Repr::default();
-    let ret_ptr = unsafe { mem::transmute(&mut ret) };
+    let ret_ptr = unsafe {
+        mem::transmute::<&mut <R as crate::conv::FromRepr>::Repr, red4ext_sys::interop::VoidPtr>(
```
Comment (jac3km4) [inline red4ext/src/invocable.rs:156]:
```suggestion
        mem::transmute::<&mut R::Repr, VoidPtr>(
```

**Pull request merged**